code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -34,7 +34,7 @@
 	if(isliving(AM))
 		var/mob/living/L = AM
 		if(L.stat <= stat_affected)
-			if((!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && affects_servants)) && L.mind)
+			if((!is_servant_of_ratvar(L) || (is_servant_of_ratvar(L) && affects_servants)) && L.mind && (!isdrone(L) || istype(L, /mob/living/simple_animal/drone/cogscarab)))
```

Comment:
We want it to continue if the target is a scarab, because those are supposed to serve ratvar.